encoding/v2/rs/backend/rs_backend.go

```diff
@@ -1,14 +1,15 @@
 package backend
 
 import (
+	"context"
 	"github.com/Layr-Labs/eigenda/encoding/v2/rs/backend/gnark"
 	"github.com/Layr-Labs/eigenda/encoding/v2/rs/backend/icicle"
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
 // Proof device represents a device capable of computing reed-solomon operations.
 type RSEncoderBackend interface {
-	ExtendPolyEval(coeffs []fr.Element) ([]fr.Element, error)
+	ExtendPolyEvalV2(ctx context.Context, coeffs []fr.Element) ([]fr.Element, error)
```

Comment:
why did we rename it to V2?

Reply:
Just a convenience since otherwise the IDE shows the encoding/v1 implementations as well. I did the same thing on the prover interface already.